### PR TITLE
Add Dutch language support

### PR DESCRIPTION
**Pull request by YannickWhorst**
## Checklist

Please put "X" in the below checkboxes that apply:

- [ ] If committing a change of production code, I have tested it in different browsers (Chrome, Firefox, Safari). 
- [X] If committing a new feature, I have first submitted an issue (Please note: you are free to open PRs for non-issued features, but opening an issue increases your chance of a successful PR). 
- [ ] If committing a new feature, I have also written the appropriate tests for it. 
- [X] I have tried to build the feature in a way, that it does not cause any breaking changes for others (unless 
  agreed upon in an issue). 

## This PR solves the following problem**. 
This PR solves the issue as described in #280 

## How to test this PR**. 
Set the locale in the datepicker or calendar config to `nl-NL`, i.e. `locale: 'nl-NL',`

### CODE REVIEW

**Comment by tomosterlund** [comment]
Hartelijk dank for opening this PR @YannickWhorst. There's a merge conflict now since I just merged the Portuguese translations. Let me know if you need any help resolving it

**Comment by YannickWhorst** [comment]
Geen probleem! I'll fix the merge conflicts.

File: packages/translations/src/locales/nl-NL/calendar.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export const calendarNlNL: CalendarTranslations = {
+  Today: 'Vandaag',
+  Month: 'Maand',
+  Week: 'Week',
+  Day: 'Dag',
+  events: 'gebeurtenissen',
+  event: 'gebeurtenis',
+  'No events': 'Geen gebeurtenissen',
+  'Next period': 'Volgende periode',
+  'Previous period': 'Vorige peride',
```
Comment (tomosterlund) [inline packages/translations/src/locales/nl-NL/calendar.ts:12]: Spelling differs: "periode" vs "peride", is that intentional?

File: packages/translations/src/locales/nl-NL/date-picker.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { DatePickerTranslations } from '../../types/date-picker.translations'
+
+export const datePickerNlNL: DatePickerTranslations = {
+  Date: 'Datum',
+  'MM/DD/YYYY': 'DD-MM-JJJJ',
+  'Next month': 'Vorige Maand',
+  'Previous month': 'Volgende maan',
```
Comment (tomosterlund) [inline packages/translations/src/locales/nl-NL/date-picker.ts:7]: Also here spelling differs between "Maand" and "maan". Is that intentional?

**Comment by tomosterlund** [comment]
> @tomosterlund fixed both of the inconsistencies, my merge has also readded(?) Brazilian Portuguese translations, but I have not changed anything in those files.

Don't worry about it, I always squash all commits anyway :)

**Comment by YannickWhorst** [comment]
Alright!

**Pull request merged**